test/sagemaker_tests/pytorch/training/requirements.txt

```diff
@@ -2,7 +2,7 @@ botocore>1.0,<2.0
 boto3>1.0,<2.0
 awscli>=1.27.51
 protobuf
-sagemaker==2.152.0
```

Comment:
I guess there was a bug in one of the SM versions, because of which we had to fix this version. In case the tests are passing, I think it is fine to remove the pin from SM.

Reply:
Yeah, we had to pin sagemaker, fabric + invoke, and other packages due to bugs that existed for a short period of time. Upon discovering that removing the version-pin doesn't break the tests on one of the versions, I unpinned them all, because older versions have dependencies that prevent the use of the latest pytest and pytest optimizations.